torch/export_utils.py

```diff
@@ -0,0 +1,18 @@
+from torch._export.constraints import constrain_as_size, constrain_as_value, dynamic_dim
+from torch._export.exported_program import ExportedProgram, ExportGraphSignature
+from torch._export.pass_base import ExportPassBase
+from torch._export.serde.serialize import deserialize, serialize
+from torch._export.utils import register_dataclass_as_pytree_node
+
+
+__all__ = [
+    "dynamic_dim",
+    "ExportedProgram",
```

Comment:
Can we get type annotations on all of the methods on ExportedProgram?

Reply:
This is a key part of the export API and it doesn't appear to have any docblocks on main. There needs to be documentation about how users can make use of ExportedProgram.

Reply:
In fact, most of the functions here don't appear to have docblocks, this seems like bare minimum for the export utilities.